xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -1881,6 +1863,34 @@ require(['jquery', 'xwiki-meta', 'bootstrap'], ($, xm) => {
         })
     }
 
+    /**
+     * Add click listeners on all rendering error messages to let the user read the detailed error description.
+     * If a content is passed, add click listener for errors reported in this content (useful for AJAX requests response)
+     * Otherwise make all the document's body errors expandable.
+     */
+    function makeRenderingErrorsExpandable(content) {
+        $(content || 'body').find('.xwikirenderingerror').each(function (index) {
+            let error = $(this);
+            let description = error.next(".xwikirenderingerrordescription");
+            if (description.innerHTML !== "" && description.hasClass("xwikirenderingerrordescription")) {
+                error.attr('id', 'xwikirenderingerror-' + index);
+                error.attr('role', 'button');
+                description.attr('id', 'xwikirenderingerrordescription-' + index);
+                error.attr('aria-controls', 'xwikirenderingerrordescription-' + index);
+                error.attr('aria-expanded', false);
```

Comment:
Note that you can use an object in attr, that would make the code maybe a bit more readable:
```
error.attr({
  'id': 'xwikirenderingerror-' + index,
  'role': 'button',
  ....
})
```